build.sbt

```diff
@@ -124,12 +129,8 @@ lazy val `pekko-connectors` = project
     crossScalaVersions := List() // workaround for https://github.com/sbt/sbt/issues/3465
   )
 
-TaskKey[Unit]("verifyCodeFmt") := {
```

Comment:
I have opted to replace this because the current style is overcomplicated especially when compared to `addCommandAlias`. This style made more sense in the past when we weren't checking that code was formatted on PR but this is no longer the case